usaspending_api/etl/elasticsearch_loader_helpers/delete_data.py

```diff
@@ -128,7 +126,7 @@ def get_deleted_award_ids(client, id_list, config, index=None):
                    ...]
      """
     if index is None:
-        index = f"{config['root_index']}-*"
+        index = f"{config['query_alias_prefix']}-*"
```

Comment:
I share the concern, I wasn't planning to address this in my PR since it is existing code. A mitigation is already in place: if a new index doesn't have multiple aliases then provide the index name to the function.

I will edit the function args to be clearer